dev/sparktestsupport/modules.py

```diff
@@ -526,6 +526,7 @@ def __hash__(self):
         "pyspark.sql.tests.pandas.test_pandas_grouped_map",
         "pyspark.sql.tests.pandas.test_pandas_grouped_map_with_state",
         "pyspark.sql.tests.pandas.test_pandas_map",
+        "pyspark.sql.tests.pandas.test_pandas_transform_with_state",
```

Comment:
Gosh, I didn't realize that pyspark test suite should be added into here. Does this mean we haven't run this test suite in previous PRs?

Reply:
Yeah unfortunately that's the case, good thing is that we made sure the test pass locally for previous PRs, so only minor changes are needed.